meteor/sofie-server/data-access/repositories/test/mongo-segment-repository.spec.ts

```diff
@@ -42,15 +43,20 @@ describe(`${MongoSegmentRepository.name}`, () => {
 			const mongoConverter: MongoEntityConverter = mock(MongoEntityConverter)
 			const partRepository: PartRepository = mock<PartRepository>()
 			const rundownId: string = 'someRundownId'
-			const segments = [
+			const segments: MongoSegment[] = [
 				createSegment({ rundownId: rundownId }),
 				createSegment({ rundownId: rundownId }),
 				createSegment({ rundownId: rundownId }),
 			]
+			const entitySegments: Segment[] = [
```

Comment:
If you renamed `segments` to `mongoSegments` (which is what they are), then you can renamed `entitySegments` to `segments`.

This comment applies to all places where you have done this.

Reply:
Has been renamed.